curriculum/test/test-challenges.js

```diff
@@ -160,9 +174,14 @@ function populateTestsForLang({ lang, challenges }) {
       describe(challenge.title || 'No title', function() {
         it('Common checks', function() {
           const result = validateChallenge(challenge);
+          const invalidBlock = validateBlock(challenge);
+
           if (result.error) {
             throw new AssertionError(result.error);
           }
+          if (challenge.challengeType !== 7 && invalidBlock) {
+            throw new AssertionError(invalidBlock);
```

Comment:
```suggestion
            throw new Error(invalidBlock);
```
I doubt error type really matters, but just in case.